packages/InteractiveProfilingTool-Tests.package/IPTModelTestCase.class/instance/testBuildButton.st

```diff
@@ -0,0 +1,4 @@
+as yet unclassified
+testBuildButton
+
+	self assert: 'PluggableButtonSpec' equals: self buildButton className
```

Comment:
Instead of testing if the method returns a spec, this should test if the builder is able to build a `PluggableButtonMorph` using the spec without errors.